rfcs/0018-glides-trainsheet-events.md

```diff
@@ -141,15 +106,37 @@ In order to reduce event duplication, a Trip Key is used to identify both added
 
 ## Events
 
+### Editors Changed
+Operator sign in and out of editing specific trainsheets. This event publishes changes to who is editing. Since one action by an inspector might result in multiple changes, for example if they sign out of one location and in to another, the changes are published in a list.
+
+In practice, we limit each location to have only one editor, and limit each editor to only one location, but that rule may or may not change and this data in this event doesn't guarantee that.
+
+Note that the author may or may not match the editors in `changes`, since inspectors can change whether other people are editing. For example, when an inspector takes over editing at a location, they may sign out the previous editor. Then they would issue a `stop` change for the previous editor and a `start` change for themself.
+
+Event type: `com.mbta.ctd.glides.editors-changed.v1`
+Fields in the event:
+- `author` (Author): the inspector who made the changes
+- `changes` (array of EditorChange): a list of start and stop editing events
+
+#### EditorChange
+Fields in the object:
+- `type` (string `"start"`|`"stop"`): Whether the editor started or stopped editing.
+- `location` (Location): the location the editor started or stopped managing.
+- `emailAddress` (string): the e-mail of the editor.
+- `badgeNumber` (string, optional): the badge number of the editor.
+
+
 ### Operator signed in
 At the start of their shift, operators need to confirm that they are fit-for-duty and do not have any electronic devices. They currently do this by physically signing a paper trainsheet: in the future, they will do this digitally.
 
 Event type: `com.mbta.ctd.glides.operator_signed_in.v1`
 Fields in the event:
 - `author` (Author): the inspector who signed in the operator
-- `operator` (Operator): the operator who signed in
+- `operator` (string): the badge number of the operator who signed in
```

Comment:
I wasn't sure about this. I was trying to streamline things to make it a little flatter and the documentation shorter. If somebody wants operator names, is this really the right way to get it, or should they get their own badge/name list. (That doesn't have the same problems as requiring clients to get their own schedule, since it doesn't have to stay in sync with Glides.) We should discuss it today.

Reply:
Reverted back to using the Operator object with a single field.